pom.xml

```diff
@@ -165,7 +165,7 @@
 		<http.builder.version>1.0.4</http.builder.version>
 		<jaxb.api.version>2.3.1</jaxb.api.version>
 		<jaxb.core.version>3.0.2</jaxb.core.version>
```

Comment:
@sumerjabri We should also update `jaxb.core.version` to `4.0.4` since jaxb-impl has a dependency on jaxb-core